infra/feast-operator/config/samples/v1alpha1_featurestore_tls_disable.yaml

```diff
@@ -0,0 +1,20 @@
+apiVersion: feast.dev/v1alpha1
```

Comment:
maybe change filename to include openshift? 
`v1alpha1_featurestore_openshift_tls_disable.yaml` ??

or maybe the better thing would be a comment at the top of the file explaining why this might be needed? because the user shouldn't need this in a typical k8s cluster